statics/src/main/java/com/codeborne/selenide/Selenide.java

```diff
@@ -911,6 +912,23 @@ public static Clipboard clipboard() {
     return getSelenideDriver().getClipboard();
   }
 
+  /**
+   * Provide access to Chromium DevTools.
```

Comment:
What about Firefox?

Reply:
According the Mozilla docs, Firefox supports CDP only subset of CDP protocol. Actually I haven't seen any CDP commands that works in Firefox.
As I see tools like Playwright patches Firefox sources and distribute its own Firefox binaries versions. I believe we can't tell that our API will support Firefox out of the box without additional impact to research how to allow Firefox actually works with CDP. This PR is a first implementation and Firefox support is out of scope.